makerspace/templates/tools/admin_tool.html

```diff
@@ -0,0 +1,90 @@
+{% extends "web/base.html" %}
+{% load staticfiles %}
+{% load i18n %}
+{% load news_tags %}
+
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "news/css/admin.css" %}">
```

Comment:
Why is the news css used?

Reply:
If the same rules are used in both tools and news, you probably want to create a separate file with the common rules under `web/static/`, and specific rules in each app. If not changes in news could cause problems with tools later